test/integration/features/provision_various_versions.feature

```diff
@@ -0,0 +1,29 @@
+@provision-various-versions @openshift @minishift-only
+Feature: Provision all major Openshift versions
+  As a user I can provision major versions of openshift
+
+Scenario Outline:: Provision all major OpenShift versions
```

Comment:
There is an extra colon, also the Scenario Outline should be indented by 2 spaces.